frontend/src/styleguide/views/Styleguide.vue

```diff
@@ -2,11 +2,23 @@
   <h1 class="text-gray-700 text-xl font-bold mb-6">
     Styleguide
   </h1>
+
+  <GoIcons :name="icons.PENCIL" class="h-5 w-5"></GoIcons>
+  <GoIcons :name="icons.TRASH" type="outline" class="h-5 w-5"></GoIcons>
+  <GoIcons :name="icons.ACADEMIC_CAP" type="outline" class="h-10 w-10 transform rotate-180"></GoIcons>
 </template>
 
 <script lang="ts">
+import GoIcons from '/@components/Icons/GoIcons.vue'
+import { icons } from '/@components/Icons/icons'
 import { defineComponent } from 'vue'
 
-
-export default defineComponent({})
+export default defineComponent({
+  components : {GoIcons},
+  setup() {
+    return {
+      icons
```

Comment:
I see what you want to do, but this goes back to the same problem we had before, which is having to import something on every template to use icons.

We don't really need the Enum.

Reply:
had the same realization